client/state/selectors/get-current-query-arguments.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ *
+ * @format
```

Comment:
My understanding is that `@format` should be on its own line.

Same thing in the other selectors.

Reply:
I believe the original codemod added it to the first comment, then another codemods replaced that https://github.com/Automattic/wp-calypso/pull/19711

I don't mind using the new syntax 👍